specs/MultiProfile.md

```diff
@@ -297,6 +413,13 @@ namespace Microsoft.Web.WebView2.Core
 
         String ProfilePath { get; };
     }
+
+    runtimeclass CoreWebView2Profile2 : CoreWebView2Profile
```

Comment:
This isn't what it should look like for additions to a runtimeclass. It should look like a new [interface_name(...)] block inside the existing CoreWebView2Profile runtimeclass. If you run build-apiwriter.ps1 you can see your additions in the generated Microsoft.Web.WebView2.Core.idl file.

Reply:
Thanks Dave! It's the first time for me to know where the runtimeclass comes from. By running `powershell .\third_party\edge_webview2\win\webview2_api_writer\dotNetAPIWrapper\build-apiwriter.ps1 -BuildTarget WinRT -Configuration Debug -Architecture x64 -PackageKind Internal`, I got some code like below in Microsoft.Web.WebView2.Core.idl file:
```
runtimeclass CoreWebView2Profile
{
    // ICoreWebView2StagingProfile2 members
    Boolean IsPasswordAutosaveEnabled { get; set; };
    
    Boolean IsGeneralAutofillEnabled { get; set; };
    
    [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2ExperimentalProfile")]
    {
        // ICoreWebView2ExperimentalProfile members
        String ProfileName { get; };
    
        Boolean IsInPrivateModeEnabled { get; };
    
        String ProfilePath { get; };
    }
}
```
It seems that only `ProfileName`, `IsInPrivateModeEnabled` and `ProfilePath` are in the block `[interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2ExperimentalProfile")]`, while `IsPasswordAutosaveEnabled` and `IsGeneralAutofillEnabled` are outside any interface block. I'm not sure is there any problems about it, so may need your help to review and confirm. Thanks!

Reply:
Oh oops, that's because of the interface isn't public yet. Once the experimental profile APIs become public and yours become public (which is what we want to show in these documents - what we expect it to look like when public) it should look like:
```
runtimeclass CoreWebView2Profile
{
    String ProfileName { get; };    
    Boolean IsInPrivateModeEnabled { get; };    
    String ProfilePath { get; };

    [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile2")]
    {
        Boolean IsPasswordAutosaveEnabled { get; set; };
        Boolean IsGeneralAutofillEnabled { get; set; };    
    }
}
```

Reply:
Thanks for clarification! Document updated.